tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -18,6 +18,23 @@
   <mat-icon svgIcon="notifications_none_24px"></mat-icon>
 </button>
 
-<mat-menu #menu="matMenu">
-  <div mat-menu-item>notification 1</div>
+<mat-menu #menu="matMenu" class="notification-menu">
+  <div>
+    <div *ngFor="let notification of notifications" class="menu-item">
+      <div>
+        <mat-icon
+          svgIcon="{{ notification.icon }}"
+          class="category-icon"
+        ></mat-icon>
+        <span>{{ notification.category }}</span>・
+        <span>{{ notification.dateInMs | date:'longDate'}}</span>
+        <span></span>
+      </div>
+      <div class="content-wrapper">
+        <span class="title">{{ notification.title }}</span><br />
+        <ul class="content" [innerHTML]="notification.content"></ul>
```

Comment:
is `content` guaranteed to have `li`? If not, it would violate the HTML structure right? Just remove `ul`?

Reply:
you're right, `li` is not necessary. replace with `div`?